build.gradle

```diff
@@ -14,13 +14,11 @@ repositories {
 }
 
 dependencies {
```

Comment:
I sometimes encountered

```General error during semantic analysis: Unsupported class file major version 63```

when removing certain dependencies.  This is due to incompatible Gradle and JVM versions.  Gradle 6.4 is compatible with JVM 14- (https://docs.gradle.org/current/userguide/compatibility.html)